<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="PyDMTimePlot" name="timeplot">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="curves">
      <stringlist/>
     </property>
     <property name="bufferSize">
      <number>60</number>
     </property>
     <property name="updatesAsynchronously">
      <bool>true</bool>
     </property>
     <property name="timeSpan">
      <double>30.000000000000000</double>
     </property>
     <property name="updateInterval">
      <double>0.500000000000000</double>
     </property>
     <property name="autoRangeY">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Channels</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Available Signals:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="signal_combo"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMDrawingRectangle" name="color">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>
    A widget with a rectangle drawn in it.
    This class inherits from PyDMDrawing.

    Parameters
    ----------
    parent : QWidget
        The parent widget for the Label
    init_channel : str, optional
        The channel to be used by the widget.
    </string>
        </property>
        <property name="brush" stdset="0">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </property>
        <property name="penStyle" stdset="0">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="penWidth" stdset="0">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="create_button">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="live_channels">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>162</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTimePlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.timeplot</header>
  </customwidget>
  <customwidget>
   <class>PyDMDrawingRectangle</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.drawing</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
